packages/yourloops/layout/main-layout.tsx

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
There is already a layout folder in components folder, maybe we can move `dashboard-layout` and `menu-layout` in this new folder.